src/data/nat/totient.lean

```diff
@@ -39,6 +39,40 @@ lemma totient_le (n : ℕ) : φ n ≤ n :=
 calc totient n ≤ (range n).card : card_filter_le _ _
            ... = n              : card_range _
 
+-- TODO move next to filter_subset_filter
+lemma filter_subset_filter_of_implies (s : finset ℕ) (p q : ℕ → Prop) [decidable_pred p] [decidable_pred q] (h : ∀ n , p n -> q n) : s.filter p ⊆ s.filter q :=
```

Comment:
Do we have a lemma of the form `mono s.filter`?

Reply:
Good point that this is an instance of monotonicity. I was not able to find such a lemma just now.

Reply:
#8842